projects/jupyter-server-ydoc/jupyter_server_ydoc/handlers.py

```diff
@@ -137,6 +137,9 @@ def exception_logger(exception: Exception, log: Logger) -> bool:
                         exception_handler=exception_logger,
                     )
 
+                    # Listen for the changes in GlobalAwareness to update users
+                    self.room.awareness.observe(self._on_global_awareness_event)
```

Comment:
This room is not necessarily the global awareness. Should we check that the room ID is `"JupyterLab:globalAwareness"` before observing awareness changes?